app/models/bike.rb

```diff
@@ -158,6 +160,38 @@ def organization(org_or_org_id)
       organization = org_or_org_id.is_a?(Organization) ? org_or_org_id : Organization.friendly_find(org_or_org_id)
       includes(:bike_organizations).where(bike_organizations: { organization_id: organization.id })
     end
+
+    # Bikes in a `held` state are stolen bikes that have a counterpart record
```

Comment:
I think `held` is actually just the state of an abandoned bike after it is registered (you register an abandoned bike, and then you hold that abandoned bike). So I don't think this definition works correctly, and I think it will be confusing down the line when we're dealing with external searching.

Reply:
Ah thanks. Agreed. I think I misunderstood your previous comment about the `held`  and `possibly_found` states. Does `Bike.possibly_found` seem more intuitive?

Reply:
Yes! 👍💯